build/test-startupscriptgenerators.sh

```diff
@@ -4,20 +4,27 @@
 # Licensed under the MIT license.
 # --------------------------------------------------------------------------------------------
 
-set -ex
+set -e
 
 declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
-declare -r GEN_DIR="$REPO_DIR/src/startupscriptgenerator/"
+declare -r GEN_DIR="$REPO_DIR/src/startupscriptgenerator/src"
+
 # When volume mounting a directory from the host machine, we host it as a readonly folder because any modifications by a
 # container in that folder would be owned by 'root' user(as containers run as 'root' by default). Since CI build agents
 # run as non-root cleaning these files would be a problem. So we copy the mounted directory in the container
 # to a different directory within the container itself and run tests on it.
-declare -r GEN_DIR_CONTAINER_RO="/src"
-declare -r GEN_DIR_CONTAINER="/go/src/startupscriptgenerator"
-declare -r MODULE_TO_TEST="startupscriptgenerator/..."
+declare -r GEN_DIR_CONTAINER_RO="/startupscriptgenerator"
+declare -r GEN_DIR_CONTAINER="/go/src"
```

Comment:
So in a go container, now the structure would need to mimic what our local workspace looks like:
So if your local path was `c:\oryx\src\startupscriptgenerator` (under which you would have `src`, `bin`, `pkg` as child folders), then the path in the container would be `/go` (under which you would have `src`, `bin`, `pkg` as child folders)